src/main/java/com/mael/payroll/paymentSchedules/BiweeklySchedule.java

```diff
@@ -1,11 +1,24 @@
 package com.mael.payroll.paymentSchedules;
 
 import java.time.LocalDate;
+import java.time.temporal.WeekFields;
+import java.util.Locale;
+
+import static java.time.DayOfWeek.FRIDAY;
 
 public class BiweeklySchedule implements PaymentSchedule {
 
     @Override
     public boolean isPayDay(LocalDate date) {
-        return false;
+        return isFriday(date) && isBiweekly(date);
+    }
+
+    private boolean isFriday(LocalDate date) {
+        return date.getDayOfWeek() == FRIDAY;
+    }
+
+    private boolean isBiweekly(LocalDate date) {
+        int weekOfYear = date.get(WeekFields.of(Locale.getDefault()).weekOfWeekBasedYear());
```

Comment:
The value on the right hand side could be extracted in a separate method in order to have `isBiweekly()` a bit smaller.